ACME documentation improvement

peschmae opened the pull request:
Adds documentation how to reuse an ACME account.

Additionally also adds the configuration parameters, to the configuration documentation.

Fixes #3149

hosseinsh commented on docs/production/index.rst:
```diff
@@ -503,3 +503,47 @@ The following must be added to the config file to activate the pinning (the pinn
+    }
+
+
+Using `python-jwt` converting a existing private key in PEM format is quite easy::
```
`an existing`

hosseinsh commented on docs/production/index.rst:
```diff
@@ -503,3 +503,47 @@ The following must be added to the config file to activate the pinning (the pinn
+
+    {"body": {}, "uri": "https://acme-staging-v02.api.letsencrypt.org/acme/acct/<ACCOUNT_NUMBER>"}
+
+The uri can be retrieved from the ACME create account endpoint, when trying to create a new account, using the existing key.
```
```
The URI can be retrieved from the ACME create account endpoint when creating a new account, using the existing key.
``

hosseinsh reviewed (commented):
thanks for the PR. minor suggestions

peschmae commented:
Thanks for the feedback. I've integrated your suggestions

hosseinsh commented:
@peschmae thanks for the quick workaround. I don't seem to have the permissions, to update the branch to latest version, before being able to merge it.

peschmae commented:
@hosseinsh That's probably because I switched over from my private fork, to an organizational one. Need to check that tomorrow.

For now, I've rebased my branched to the master, and it should be up to date again

hosseinsh commented:
Thanks @peschmae for the PR!
Documentation is certainly an area, we need to improve.
